test/e2e/framework/pv_util.go

```diff
@@ -295,53 +354,72 @@ func CreatePVsPVCs(numpvs, numpvcs int, c clientset.Interface, ns string, pvConf
 
 	// create pvs and pvcs
 	for i = 0; i < pvsToCreate; i++ {
-		pv, pvc = CreatePVPVC(c, pvConfig, pvcConfig, ns, false)
+		if pv, pvc, err = CreatePVPVC(c, pvConfig, pvcConfig, ns, false); err != nil {
+			return pvMap, pvcMap, err
```

Comment:
That's a good question. So far, all of the callers of CreatePVsPVCs() call `Expect()` when an err is returned. If >0 PVs have been created and then an error occurs, these PVs will be orphaned (PVCs will be deleted when the namespace is deleted.) I am thinking that CreatePVsPVCs() should clean up created resources (PVs and PVCs) when it detects an error. @copejon please feel free to comment.